csp-mixin/signals/gcpceOverview.libsonnet

```diff
@@ -0,0 +1,284 @@
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableReceivedBytes: {
+        name: 'tableReceivedBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_network_received_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableReadBytes: {
+        name: 'tableReadBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_read_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      tableWriteBytes: {
+        name: 'tableWriteBytes',
+        type: 'raw',
+        sources: {
+          stackdriver: {
+            expr: 'sum by (instance_name, job, project_id)(rate(stackdriver_gce_instance_compute_googleapis_com_instance_disk_write_bytes_count{job!="",job=~".+",project_id=~".+"}[$__rate_interval]))',
+            legendCustomTemplate: '',
+            step: '1m',
+          },
+        },
+      },
+      // end table
+
+      memoryUtilization: {
+        name: 'Memory Utilization',
+        description: 'Memory Utilization',
+        type: 'raw',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'sum(stackdriver_gce_instance_compute_googleapis_com_instance_memory_balloon_ram_size{instance_name=~"$instance_name"} or on() vector(0)) + sum(stackdriver_gce_instance_compute_googleapis_com_guest_memory_bytes_used{instance_name=~"$instance_name"} OR on() vector(0))',
+            legendCustomTemplate: 'Total Memory Capacity',
+          },
+        },
+      },
+
+      memoryUsed: {
+        name: 'Memory Utilization2',
+        description: 'Memory used',
+        type: 'raw',
+        unit: 'bytes',
+        sources: {
+          stackdriver: {
+            expr: 'sum(stackdriver_gce_instance_compute_googleapis_com_instance_memory_balloon_ram_used{instance_name=~"$instance_name"} OR on() vector(0)) + sum(stackdriver_gce_instance_compute_googleapis_com_guest_memory_bytes_used{instance_name=~"$instance_name", state!="free"} OR on() vector(0))',
+            legendCustomTemplate: 'Total Memory Used',
+          },
+        },
+      },
+
+      packetsSent: {
+        name: 'Total packets count sent/received',
+        description: 'Count of packets sent/received over the network. Sampled every 60 seconds',
+        type: 'counter',
+        unit: 'short',
```

Comment:
For some reason, this panel is the only one where the unit is not applied. any idea?

Reply:
Apparently, this is getting overwritten by the [generateUnits](https://github.com/grafana/jsonnet-libs/blob/master/common-lib/common/signal/utils.libsonnet#L49-L61) function of the signals library, specifically [here](https://github.com/grafana/jsonnet-libs/blob/master/common-lib/common/signal/utils.libsonnet#L56).

Candidly... I'm not sure why.. @v-zhuravlev may be able to shed some light on that 🤔

Reply:
`short` means no units basicaly... no?

Reply:
If I use short in the UI, I get the same result as `sishort`, but here it's not the same.
I changed the unit to `sishort` and now I get the result I wanted. Thanks!
https://github.com/grafana/jsonnet-libs/pull/1336/commits/7b2e3a357f88beb10c2e9ec581226f63331ba80b